bin/oref0-calculate-iob.js

```diff
@@ -43,13 +44,24 @@ if (!module.parent) {
   var profile_data = require(cwd + '/' + profile_input);
   var clock_data = require(cwd + '/' + clock_input);
 
+  var autosens_data = null;
+  if (autosens_input) {
+    try {
+        var autosens_data = require(cwd + '/' + autosens_input);
+    } catch (e) {}
```

Comment:
i think that at least an errors should be logged if autosens data cannot be parsed. Python Zen: "Errors should never pass silently. Unless explicitly silenced.". I don't see a need for explicitly silincing this error.

Reply:
Anything we log here shows up dozens or hundreds of times in the pump-loop.log.  We instead check for the validity of autosens.json in oref0-determine-basal and warn the user there if it's invalid.

More generally, I usually don't trust advice containing the words "never" or "always", so I don't consider absolute dictates from Python Zen to be very strong arguments in favor of doing something.  We need actual concrete reasons for why doing something outweighs the costs.